diff --git a/node_modules/alertifyjs/build/alertify.js b/node_modules/alertifyjs/build/alertify.js
index 5d620ff..029b734 100644
--- a/node_modules/alertifyjs/build/alertify.js
+++ b/node_modules/alertifyjs/build/alertify.js
@@ -28,18 +28,18 @@
         basic:false,
         closable:true,
         closableByDimmer:true,
-        invokeOnCloseOff:false,
+        invokeOnCloseOff:true,
         frameless:false,
         defaultFocusOff:false,
         maintainFocus:true, //global default not per instance, applies to all dialogs
-        maximizable:true,
+        maximizable:false,
         modal:true,
-        movable:true,
+        movable:false,
         moveBounded:false,
         overflow:true,
         padding: true,
-        pinnable:true,
-        pinned:true,
+        pinnable:false,
+        pinned:false,
         preventBodyShift:false, //global default not per instance, applies to all dialogs
         resizable:true,
         startMaximized:false,
@@ -65,8 +65,8 @@
             }
         },
         glossary:{
-            title:'AlertifyJS',
-            ok: 'OK',
+            title:'',
+            ok: 'Delete',
             cancel: 'Cancel',
             acccpt: 'Accept',
             deny: 'Deny',
